### PR TITLE
Respond to 'uuid please' with a UUID

### PR DESCRIPTION
**Pull request by blundell89**
Open to comments about how we should request one

### CODE REVIEW

File: src/uuid.coffee [hunk 1]
```diff
@@ -7,7 +7,8 @@
 # Author:
 #   Lee Blundell <leeblundell16@gmail.com>
```
Comment (Codesleuth) [inline src/uuid.coffee:11]: Add help :)
https://hubot.github.com/docs/scripting/#documenting-scripts

File: src/uuid.coffee [hunk 2]
```diff
@@ -7,7 +7,8 @@
...
 
+uuidFactory = require('node-uuid')
```
Comment (Codesleuth) [inline src/uuid.coffee]: Call it just `uuid`?

**Comment by Codesleuth** [comment]
`.travis.yml` should probably be:

``` yaml
language: node_js
node_js:
  - node
  - 4.4.5

script:
  - npm run build
  - npm test

cache:
  directories:
    - node_modules
```

**Comment by terrawheat** [comment]
Curious, why coffee script? Just because?

**Comment by blundell89** [comment]
@terrawheat used Yeoman to generate the script (https://github.com/desmondmorris/generator-hubot-script) and it is CoffeeScript by default

**Pull request merged**